src/main/java/frc/team2412/robot/RobotMap.java

```diff
@@ -2,154 +2,220 @@
+    public static DigitalInput back = new DigitalInput(backSensorID);
+    public static DigitalInput backMid = new DigitalInput(backMidSensorID);
+    public static DigitalInput backInner = new DigitalInput(backInnerSensorID);
+    public static DigitalInput frontInner= new DigitalInput(frontInnerSensorID);
+    public static DigitalInput frontMid = new DigitalInput(frontMidSensorID);
+    public static DigitalInput front = new DigitalInput(frontSensorID);
+
+    private static class IndexIntakeSelector {
+        IndexIntakeSelector() {
+            indexFrontMotor = tryGetMotor(IndexIntakeModule.ONE.getIndexCANID());
+            indexBackMotor = tryGetMotor(IndexIntakeModule.TWO.getIndexCANID());
+            intakeFrontMotor = tryGetMotor(IndexIntakeModule.ONE.getIntakeCANID());
+            intakeBackMotor = tryGetMotor(IndexIntakeModule.TWO.getIntakeCANID());
+            if (indexFrontMotor == null) {
+                indexFrontMotor = tryGetMotor(IndexIntakeModule.THREE.getIndexCANID());
+                intakeFrontMotor = tryGetMotor(IndexIntakeModule.THREE.getIntakeCANID());
+            } else if (indexBackMotor == null) {
+                indexBackMotor = tryGetMotor(IndexIntakeModule.THREE.getIndexCANID());
+                intakeBackMotor = tryGetMotor(IndexIntakeModule.THREE.getIntakeCANID());
+            }
+        }
+
+        private CANSparkMax tryGetMotor(int motorId) {
+            try {
+                return new CANSparkMax(motorId, MotorType.kBrushless);
+            } catch (Exception ignored) {
+                return null;
+            }
+        }
+    }
+
+    public static IndexIntakeSelector indexSelector = new IndexIntakeSelector();
+
+    // INDEXER CONTROLS THESE NOT INTAKE FYI
+    public static DigitalInput intakeFront = new DigitalInput(intakeFrontSensorID);
+    public static DigitalInput intakeBack = new DigitalInput(intakeBackSensorID);
+    public static final int exampleID = 1;
+
+    // Turret Subsystem
+    // ------------------------------------------------------------------------------
+    public static final int turretMotorID = 1;
+    public static WPI_TalonSRX turretMotor = new WPI_TalonSRX(turretMotorID);
+
+    // Flywheel subsystem
+    // ------------------------------------------------------------------------------
+    public static final int flywheelLeftMotorID = 0;
+    public static final int flywheelRightMotorID = 2;
+
+    public static CANSparkMax flywheelLeftMotor = new CANSparkMax(flywheelLeftMotorID, MotorType.kBrushless);
+    public static CANSparkMax flywheelRightMotor = new CANSparkMax(flywheelRightMotorID, MotorType.kBrushless);
+
+    // Hood Subsystem
+    // -----------------------------------------------------------------------------
+    public static final int HOOD_SERVO_PORT = 1;
+    public static Servo hoodServo = new Servo(HOOD_SERVO_PORT);
+
+    // INTAKE SUBSYSTEM
+
+    // Intake DoubleSolenoid Ports
+    public static final int INTAKE_FRONT_UP_PORT = 1;
```

Comment:
These need to be unique, and there are now only two ports.  Consider 3 & 4 here.